projects/aca-playwright-shared/src/page-objects/components/aca-header.component.ts

```diff
@@ -46,8 +48,27 @@ export class AcaHeader extends BaseComponent {
     await this.moreActionsButton.click();
   }
 
+  async clickMoreActions(): Promise<void> {
+    await this.toolbarMoreActions.waitFor({ state: 'attached' });
+    await this.toolbarMoreActions.click();
+  }
+
   async clickCreateFolderFromTemplate(): Promise<void> {
     await this.createButton.click();
     await this.matMenu.createFolderFromTemplate.click();
   }
+
+  async verifyToolbarPrimaryActions(expectedToolbarPrimary: string[]): Promise<void> {
```

Comment:
Hmm do we have everything properly configured? Intellij shows that all these functions are unused but when I search for them they are used, see below screenshots: 

1. You can see above each function there is 'no usages' text: 
<img width="1109" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/38f1210a-414e-437a-9155-b6d3c4a9bb5c">

2. If I search for any of these function then it finds some: 
<img width="1365" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/67883a5a-747e-4a60-baf9-053ffe60fb5c">

3. If I hover any usages then it shows me following: 
<img width="888" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/e4f415c9-b2ad-4d4e-8db9-abcee3714a0a">


Also imports seem showing some errors: 
<img width="1158" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/0e0b2d5f-359b-4a1f-a9bf-ab34ff69be8a">

Reply:
can you please help to check this, I am able to see same on Visual studio code.

Reply:
ok I will check if I see anything interesting what can help

Reply:
@akashrathod28 some observation. If you change path to relative instead of "@alfresco/playwright-shared" then everything seems fine: 
<img width="1071" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/86f64dc8-08ad-4163-9aba-c937ce6c9d91">
<img width="1024" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/ce6542a8-bc09-4362-bb3c-f9d00d053ce6">


No error for relative path and functions are detected. Probably something wrong with configuration for "@alfresco/playwright-shared". Not sure for now what

Reply:
@akashrathod28 I found reason, please just add following: 
<img width="1495" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/7f2d4b52-fe84-4377-8544-4800592700c7">

to tsconfig.e2e.json file for these e2es. 

NOTE: The same occurs for other e2e too so please update others tsconfig.e2e.json too 

After that should working fine: 
<img width="972" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/dee87d36-8da8-4ca5-bab7-8ab8a17e84ea">

<img width="1040" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/c85cb32e-bdc7-4f15-b9ff-3a876a18eda8">


No errors and functions are detected properly